sdks/python/apache_beam/io/gcp/pubsub.py

```diff
@@ -444,3 +449,69 @@ def display_data(self):
 
   def writer(self):
     raise NotImplementedError
+
+
+class PubSubSourceDescriptor(NamedTuple):
+  """A PubSub source descriptor for ``MultipleReadFromPubSub```"""
+  source: str
+  id_label: str = None
+  timestamp_attribute: str = None
+
+
+class MultipleReadFromPubSub(PTransform):
```

Comment:
Please add a code snippet about how to use this transform and highlight the benefit of using this transform compared to the `ReadFromPubSub`.